pkg/rescheduler/strategies/duplicates.go

```diff
@@ -21,53 +21,64 @@ import (
 	"strings"
 
 	"k8s.io/kubernetes/pkg/api/v1"
+	//TODO: Change to client-go instead of generated clientset.
 	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
 
 	"github.com/aveshagarwal/rescheduler/pkg/api"
 	"github.com/aveshagarwal/rescheduler/pkg/rescheduler/evictions"
 	podutil "github.com/aveshagarwal/rescheduler/pkg/rescheduler/pod"
 )
 
+// This strategy evicts all duplicate pods on node. A pod is said to be a duplicate of other if
+// both of them have same name, kind and are within the same namespace. As of now, this strategy won't
+// evict daemonsets, mirror pods and pods with local storages.
+
 //type creator string
 type DuplicatePodsMap map[string][]*v1.Pod
 
-func RemoveDuplicatePods(client clientset.Interface, strategy api.ReschedulerStrategy, policyGroupVersion string, nodes []*v1.Node) {
+// RemoveDuplicatePods removes the duplicate pods on node.
+func RemoveDuplicatePods(client clientset.Interface, strategy api.ReschedulerStrategy, policyGroupVersion string, nodes []*v1.Node) (int, error) {
```

Comment:
I want to keep all strategies call same (same input, no return) for now. So do no return here, see lownodeutilization for reference.

Reply:
Ok, I was thinking of having a strategies function which calls the RemoveDuplicatePods so that we can have a single signature for all the strategies but I am fine with this approach as well. I need to think of another way to find the exact number of pods on the node.

Reply:
I want to have strategy calls with same input and return types so that it'd be easier to move to a registration based approach, where when a rescheduler is created, strategies are registered first with it and then rescheduler executes them.